### PR TITLE
Revert invitation_code

### PR DESCRIPTION
**Pull request by carkod**
## Done

- [x] Reverted back to previous `/login?invitation_code=XXXX&next=/demo` format
- [x] Additional cleanup and change to production endpoint (towards making streaming work, regions are not available in staging)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/
- Turn on VPN
- Login with user without invitation code ->  "http://0.0.0.0:8043/login" it will fail
- Login with user with invitation code -> "http://0.0.0.0:8043/login?invitation_code=XXXX&next=/demo" where XXXX needs to be generated by anbox-cloud API manually.

### CODE REVIEW

**Comment by nottrobin** [comment]
Wait, what's the initial bug here?

I thought we would simply send people to `/login?invitation_code=dgsfgs&next=/demo`. It doesn't seem like we need to pass the invitation_code all the way from `/demo` do we?

File: webapp/app.py [hunk 1]
```diff
@@ -63,9 +61,14 @@ def login_required(func):
 
     def is_user_logged_in(*args, **kwargs):
```
Comment (nottrobin) [inline webapp/app.py]: I'm confused. I thought you removed this function? It doesn't seem to be used anywhere:

``` bash
robin@rt480:~/Projects/anbox-cloud.io (fix-login) $ rg is_user_logged_in
webapp/app.py
62:    def is_user_logged_in(*args, **kwargs):
74:    return is_user_logged_in
```

Comment (tbille) [reply webapp/app.py]: This is a function that is necessary to create the decorator: `@login_required` that we use for endpoints behind SSO

Comment (nottrobin) [reply webapp/app.py]: Oh sorry, I see. My bad.

**Comment by tbille** [comment]
@nottrobin The work done on the previous PR  is something that looks like this: `/demo?invitation_code=adsf`

File: webapp/app.py (hunk 1 above)
Comment (carkod) [reply webapp/app.py]: No, I wrote a comment saying that when I remove it, it returns an error. The decorator must return a function

**Pull request merged**